README.md

```diff
@@ -75,11 +102,13 @@ helps avoid over-engineering and meaningless work.
 Epinio's main goal is to make existing solutions accessible, to the application
 developer. Those solutions include:
 
-- Kubernetes
-- External Cloud Services
-- In-cluster services
 - TLS Certificate Signers
-- more
+- Service mesh integration and testing
```

Comment:
I believe that this section should clarify that this is __not__ a proper roadmap, and just themes and things we consider possible.

Reply:
Done with commit f84f601a8bc5c8f10cdb44581e2d6a09ade37fbb